sdk/storage/azure-storage-file-datalake/pom.xml

```diff
@@ -104,6 +104,12 @@
       <version>3.2.7</version> <!-- {x-version-update;cglib:cglib-nodep;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.objenesis</groupId>
+      <artifactId>objenesis</artifactId>
+      <version>3.1</version>
```

Comment:
Need to add a version tag for this in `external_dependencies.txt`, this is why analyze is failing.